usr/src/uts/common/vm/vm_page.c

```diff
@@ -5685,7 +5685,8 @@ page_migrate(
 	}
 }
 
-#define	MAX_CNT	60	/* max num of iterations */
+static uint_t reclaim_maxcnt = 60; /* max total iterations */
```

Comment:
Cool, I'll go ahead and remove `static` now. I didn't realize that was a restriction for tunables, thanks.